components/StyledInput.js

```diff
@@ -61,6 +62,7 @@ const StyledInput = styled.input`
 
   &:focus, &:hover:not(:disabled) {
     border-color: ${themeGet('colors.primary.300')};
+    
```

Comment:
```suggestion
```

Reply:
oh okay. let me fix that in a few. thanks

Reply:
Done